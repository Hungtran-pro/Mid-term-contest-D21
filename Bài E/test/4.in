10
822 9
egdhigaeeafcbgdi??dfbgffe?hecebhcciehacegcg?abcdcbheagiecchde?gdigfcdbgfibedhgdggihehbfh?ef?abbhehedbige?ffih???chacdia?h?ibfdfbf?befcacgihci?bghddiefidhbhdbhihe?aidebghaachfdba?ihaif?g?aefabdgaidfgf?g??eadegeecdbfcdgccgccdecchehheahihhd?fhch?gcifggcdbaf?ca??dbdihh?eg?badcf?ae?g?iaaiibdi?bccbb?cagiga?hcediiichgge?bfagfdfbcccdhfbdhfebebi?idheheg?da??bhdaidbccceg?cdfdhadhbfcdiigeiibiefbff?ehfcdhhi?bbfbgagidcaiedcibcaihie?ibgffhaibbebehehdfecg?giaehfhgaaccfgcdbiaggf?dafgdafce?eiffdgbdd?fafffdbg?ahadegc?chcdaeefagdibhh?defh?gc?dhei?ihbfbffccdhaidfgebfda?ifdffchachddaeeed?a?chefehdhbhefca?ffdcedae?fgc?chf?ifaafdab?aceibg?eigiddfiaiddbech?chfac??hihgcgg??ef?dfbc?bccgc?eeiagcb??g?cbbe?fcbadhagiagcfcfaghedcbfgdbbbbibeiiccchdefa?eggaiiad?bidhfcdiieifeigcaebfggbefhg?bgdgbfchfedfiaiidhd?bifadcgdeeggeebbbhi?i?ieeiggfddc??b
582 7
c?eg?ddcfdddebbcdbfcfd?gc?fbeabfadabgb?eecgagbfebdageccageceefgcbcc?cgff?acfcb?afa??fggfae?baddecfbddbbg?eb?aacfdfcdffe?edggabbefcageffgdcccfcefcfeg?dfec?fcefe?cbb?eefe?ddbfc?bbae???acfdeedeg?dceadgaa?acbfc?cgecegeebccc?cbdgcggefcdgbba?c?gcdbcfedfgae?c?cgcfaccfe?ffcbba??bbddfcaddecgaea??fbdbcfegeadge?bgfg?ecdfdcgcaf?faa?gcaad?gaeadaebdfb??eg?dgggfdgdgdad?g?cedffcadbdfab?ddcdfedab?cdegdebfeeaedaddebecabaabbeec?faabcefbcgadcdfb??acdgge?c?ddbfcgfb?bafggfeaegbdb?dedbgeegdage?bgcfegb?bbbaefdgbcb?ac??adaacaeedgfc?fg?dffdf?g?ce?agbfcafebbdc?cbdefbg?cac??bgcaecbcfadbbcgfe?g?dgbe?dbgf
875 7
gceggbdafba?bg?ed?gcdgaf??gfbff?aafbcfdafefdbadb?cacbdbbaggegaff?e?edabcaadbgdegbaddcbg?cgbdaefeda?cfdefbdf?bcdafcagcfbdgbeegc?gcd?f?bbcf?ga?cge?gbf?ddf??aedfcbgeafgedggbgd?ef?dggcbcdbda?aebgabgcff?efb?beedad?cdeegc?ffbfcbcb?ecgffa?gfcgfcffbbebcde?acgca?ffbccaea?a?ebbdg?bdcegbabdceegdgbecedb?f??ae?dgcafacdggefdffaacdbbdcegbbfe?c?dffa?ceafdagddcf?eefeeaefggabfdbbd?gef?bcab??bcbd?d??ebffabdcgcdcf??gc?eecfg?gcgcdbbbdbfgceefadag?ffe?adegadgggbbfb???gc?bgbaafbgc?cb?efdgdccacbcfcdbcag?eda?degaebgecbfb?dcfe?efbgda??gb?dbaec?ef?gga??adeg?cfca?fga?e?debg?gcbeefceefgdab?dfb?ecabagbgfee?ed?fbgbcdeggbedd?aa?afaggeggbfgegb?bgbbfde?b?dgcbfcef?aggbea?agafggddeegeceb?aff?fd?efddbffff?daagegfcdaebfb?gfagfcbaaaadgbg?c?geddgc?cac?dcdabcecddged??bc?abgcgg?a?g?b?bcbacac?agae?eagccaeeeb?fefaabbedadfcbdfbfa??fbg?ebdcfdfgf?ggaga?dbccaecffcdcbagbadcbcagcgdgdeeadgaaeab?aef?gb?fcgfbabbacgf
833 6
??????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????dddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa?????
695 10
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeejjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccchhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg?????
535 8
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb???????
586 8
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeegggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaagggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggdddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd??
518 6
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaacccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccceeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeedddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd????????????????????????????????????????????????????????????????????????????????????????
875 6
?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeedddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc?????
724 7
????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
