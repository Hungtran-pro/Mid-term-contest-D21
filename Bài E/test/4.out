3
3
3
92
1
7
0
44
72
103
